src/v2/Components/Payment/SavedCreditCards.tsx

```diff
@@ -166,7 +166,7 @@ export class CreditCard extends React.Component<
 
 export const RemoveLink = styled.div`
   text-align: right;
-
+  text-decoration: underline;
```

Comment:
This whole thing should be a `Clickable` instead. We shouldn't put onClick handlers on divs